src/utils/Utils.ts

```diff
@@ -1556,4 +1560,29 @@ export default class Utils {
   public static unescapeCsvValue(value: any): void {
     // double quotes are handle by csvToJson
   }
+
+  public static sanitizeCSVExport(data: any): any {
```

Comment:
[low] You can use 'unknown' instead of 'any' to tell the linter that you know what you are doing :-)

Reply:
the data.result is of type any :(

Reply:
hahaha, we have not yet changed it everywhere :).
If it's too many changes, forget it.
Thanks!